bucket/aws-session-manager-plugin.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "1.2.54.0",
+    "description": "This plugin helps you to use the AWS Command Line Interface (AWS CLI) to start and end sessions to your managed instances",
+    "homepage": "https://github.com/aws/session-manager-plugin",
+    "license": "Apache-2.0",
+    "url": "https://s3.amazonaws.com/session-manager-downloads/plugin/1.2.54.0/windows/SessionManagerPluginSetup.exe",
+    "hash": "91f578f1c30dc4b0ab3af642ef5c9c20586e3dd436775a9ea7c77fa4ed734813",
+    "bin": "bin\\session-manager-plugin.exe",
+    "architecture": {
+        "32bit": {
+            "installer": {
```

Comment:
Would it work without the installer script?

Reply:
I tested it and it works fine. The install has a problem that it cannot specify the installation directory. Therefore, you have to extract it.